src/mscorlib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -14,6 +14,20 @@ namespace System.Globalization
 {
     internal partial class CultureData
     {
+        private static bool s_invariantGlobalizationMode = InitGlobalizationInvariantMode();
```

Comment:
sorry I am not clear about that. I am using this bool static in many static methods. if I cache it in instance field how I am going to use it in such static methods? could you please clarify more what you mean here?